jvm/src/vyxal/gen/GenerateDocs.scala

```diff
@@ -57,4 +59,96 @@ private object GenerateDocs:
     SugarMap.trigraphs
       .map { case (key, value) => s"$key -> $value" }
       .mkString("", "\n", "\n")
+
+  def elementTable(): String =
+    val header =
+      "| Symbol | Trigraph |  Name | Keywords | Arity | Vectorises | Overloads |"
+    val divider = "| --- | --- | --- | --- | --- | --- | --- |"
+    val contents = StringBuilder()
+    val formatOverload = (overload: String) =>
+      val (args, description) = overload.splitAt(overload.indexOf("->"))
+      val newDesc = description
+        .replace("|", "\\|")
+        .replace("->", "")
+        .stripLeading()
+        .stripTrailing()
+      if args.stripTrailing() == "" then s"`$newDesc`"
+      else
+        s"`${args.stripTrailing().replace("|", "\\|").replace("->", "")}` => `$newDesc`"
+    Elements.elements.values.toSeq
+      .sortBy { elem =>
+        // Have to use tuple in case of digraphs
+        (
+          Lexer.Codepage.indexOf(elem.symbol.charAt(0)) +
+            (if "#∆øÞ".contains(elem.symbol.charAt(0)) then 400 else 0),
+          Lexer.Codepage.indexOf(elem.symbol.substring(1)),
+        )
+      }
+      .foreach(elem =>
+        if !elem.symbol.startsWith("#|") then
+          var trigraph = ""
+          SugarMap.trigraphs
+            .collect { case (tri, s) if s == elem.symbol => tri }
+            .foreach { tri => trigraph = tri }
+          var overloads = elem.overloads
+          contents ++=
+            s"| `${"\\".repeat(if elem.symbol == "`" then 1 else 0) +
+                elem.symbol.replace("|", "\\|")}` | ${trigraph} | ${elem.name
+                .replace("|", "/")} | ${elem.keywords
+                .map("`" + _ + "`")
+                .mkString(", ")} | ${elem.arity.getOrElse("NA")} | ${if elem.vectorises then ":white_check_mark:"
+              else ":x:"} | ${formatOverload(overloads.head)}\n"
+          overloads = overloads.tail
+          while overloads.nonEmpty do
+            contents ++= s"| | | | | | | ${formatOverload(overloads.head)}\n"
+            overloads = overloads.tail
+      )
+
+    val elements = header + "\n" + divider + "\n" + contents.toString
+
+    contents.setLength(0)
+
+    val modifierHeader =
+      "| Symbol | Trigraph | Name | Keywords | Arity | Description |"
+    val modiDivider = "| --- | --- | --- | --- | --- | --- |"
+
+    Modifiers.modifiers.keys
+      .zip(Modifiers.modifiers.values)
+      .toSeq
+      .sortBy((modi, _) =>
+        (
+          Lexer.Codepage.indexOf(modi.charAt(0)) +
+            (if "#∆øÞ".contains(modi.charAt(0)) then 400 else 0),
+          Lexer.Codepage.indexOf(modi.substring(1)),
+        )
+      )
+      .foreach(modi =>
```

Comment:
I'd much rather we did `.foreach { case ... =>` here instead of `modi._1` below. It's not immediately clear what `_1` is, so a variable name would help

Reply:
Shhh don't tell anyone but this was copiloted